app.rb

```diff
@@ -0,0 +1,64 @@
+require_relative 'lib/database_connection'
+require_relative 'lib/item_repository'
+require_relative 'lib/order_repository'
+
+# We need to give the database name to the method `connect`.
+DatabaseConnection.connect('shop_manager')
+
+item_repository = ItemRepository.new
+order_repository = OrderRepository.new
+
+class Application
+    def initialize(database_name, io, order_repository, item_repository)
+      DatabaseConnection.connect(database_name)
+      @io = io
+      @order_repository = order_repository
+      @item_repository = item_repository
+    end
+  
+    def run
+      puts "Welcome to the shop management program!"
+      @io.puts "\nWhat do you want to do" "\n 1 = list all shop items \n 2 = create a new item \n 3 = list all orders \n 4 = create a new order"
```

Comment:
Have you consider printing each line on a separate line to make it more clearer?